From a4672257ddc8f72d528091d20aa2242293adc17a Mon Sep 17 00:00:00 2001
From: Mingke Wang <mingke.wang@freescale.com>
Date: Thu, 19 Mar 2015 14:15:25 +0800
Subject: [PATCH 2/9] gstplaysink: don't set async of custom text-sink to false

set async to false lead to A/V sync problem when seeking.
the preroll need use GAP event instead of set async to false.

Upstream-Status: Inappropriate [i.MX specific]

Signed-off-by: Mingke Wang <mingke.wang@freescale.com>
---
 gst/playback/gstplaysink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 mode change 100644 => 100755 gst/playback/gstplaysink.c

diff --git a/gst/playback/gstplaysink.c b/gst/playback/gstplaysink.c
old mode 100644
new mode 100755
index aa43111..e670934
--- a/gst/playback/gstplaysink.c
+++ b/gst/playback/gstplaysink.c
@@ -2431,7 +2431,7 @@ gen_text_chain (GstPlaySink * playsink)
           G_TYPE_BOOLEAN);
       if (elem) {
         /* make sure the sparse subtitles don't participate in the preroll */
-        g_object_set (elem, "async", FALSE, NULL);
+        //g_object_set (elem, "async", FALSE, NULL);
         GST_DEBUG_OBJECT (playsink, "adding custom text sink");
         gst_bin_add (bin, chain->sink);
         /* NOTE streamsynchronizer needs streams decoupled */
-- 
1.9.1

